frontend/src/views/HomeView.vue

```diff
@@ -52,18 +55,6 @@ const toHousing = (): void => {
   background-color: $app-primary;
```

Comment:
Ref comment in variable.scss about updating these to use `var()`

Reply:
Changed to utilize `var()` in call cases.